SampleApps/TestApp/ContentView.swift

```diff
@@ -126,9 +129,108 @@ struct GetIdentitiesView: View {
 
 struct AdvertisingIdentifierView: View {
     @State var adIdText: String = ""
+    @State var adID: UUID?
+    @State var resultText: String = ""
+    
+    func getConsents() {
+        Consent.getConsents() { consents, error in
+            if let consents = consents {
+                print(consents)
+            } else if let error = error {
+                print("Error getting consents: \(error)")
+            }
+        }
+    }
+    
+    /// Provides the `advertisingIdentifier` for the given environment, assuming tracking authorization is provided
+    ///
+    /// Simulators will never provide a valid UUID, regardless of authorization; use the set ad ID flow to test a specific ad ID.
+    func getAdvertisingIdentifierForEnvironment() -> UUID {
+        #if targetEnvironment(simulator)
+        print("""
+            Simulator environment detected. Please note that simulators cannot retrieve valid advertising identifier
+            from the ASIdentifierManager (as specified by Apple); an all-zeros UUID will be retrieved even if
+            authorization is provided. If you want to use a specific ad ID, you can use the set ad ID flow.
+            """)
+        #endif
+        print("Advertising identifier: \(ASIdentifierManager.shared().advertisingIdentifier)")
+        return ASIdentifierManager.shared().advertisingIdentifier
+    }
+    
+    /// Requests tracking authorization from the user; prompt will only be shown once per app install, as per Apple rules
+    ///
+    /// It is possible to change tracking permissions at the Settings app level. Any change in tracking permissions will terminate the app.
+    /// It is also possible for system-wide tracking to be off but individual per-app permissions granted.
+    /// If "Allow Apps to Request to Track" at the system level was on and is turned off, a system prompt appears asking if previously provided individual per-app tracking permissions should be kept as-is or all turned off
+    func requestTrackingAuthorization() {
+        /// Based on Apple documentation for `ASIdentifierManager.shared().advertisingIdentifier`, iOS 14.5+ is the cutoff for required permissions request to use IDFA
+        /// however, based on testing with iOS 14.0.1 simulator, `isAdvertisingTrackingEnabled` is false on fresh app install, even if device has device level tracking enabled; prompt is never given and app does not show up in Privacy -> Tracking app list
```

Comment:
Is this additional info required since you already cover the two cases below?

Reply:
I was thinking about giving more background information regarding the iOS version check being 14+ instead of 14.5+(since the change is supposed to affect only 14.5+), in case of any confusion on why 14+ is being used; I can move this additional detail into the discussion section of the method-level documentation, or if you think it is unnecessary remove it altogether